deployment/gcp/flyte_helm_generated.yaml

```diff
@@ -1488,6 +1533,77 @@ spec:
             name: datacatalog-config
           name: config-volume
 ---
+# Source: flyte/templates/flytescheduler/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flytescheduler
+  namespace: flyte
+  labels: 
+    app.kubernetes.io/name: flytescheduler
+    app.kubernetes.io/instance: flyte
+    helm.sh/chart: flyte-v0.1.10
+    app.kubernetes.io/managed-by: Helm
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      app.kubernetes.io/name: flytescheduler
+      app.kubernetes.io/instance: flyte
+  template:
+    metadata:
+      annotations:
+        configChecksum: "2e0464032fd6a2ea97c18c9f18328780d599a1df9442029b520e0dceb516280"
+      labels: 
+        app.kubernetes.io/name: flytescheduler
+        app.kubernetes.io/instance: flyte
+        helm.sh/chart: flyte-v0.1.10
+        app.kubernetes.io/managed-by: Helm
+    spec:
+      initContainers:
+      - command:
```

Comment:
Do we not need to run migrations here too? this can be added in the future...

Reply:
No, currently flyteadmin handle's the migration...When we have a separate DB for flytescheduler then we can run migration here 
CC: @pmahindrakar-oss

Reply:
Hi @EngHabu , we are relying on the admin running the migration script for now but it can easily be decoupled to run as a separate init container for scheduler specific tables in the future